airbyte-integrations/connectors/source-trustpilot/source_trustpilot/spec.yaml

```
@@ -0,0 +1,41 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Trustpilot Spec
+  description: Trustpilot Spec schema
+  type: object
+  required:
+    - app_name
+    - start_date
+  additionalProperties: true
+  properties:
+    app_name:
+      title: App Name
+      description: This is Your app or company name, for which You need reviews from Trustpilot. For example, in https://www.trustpilot.com/review/{MY_APP_NAME}, where MY_APP_NAME is the value of Your App Name.
+      type: string
+      examples:
+        - free-now.com
+        - limos4less.com
+        - tempuslogix.com
+        - adkos.com
+    start_date:
+      title: Start Date
+      description: This is start date, from which You need reviews from Trustpilot
+      type: string
+      pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}$
+      examples:
+        - YYYY-MM-DD
```

```suggestion
        - YYYY-MM-DD
```
This is not an example. You can transfer this information to description.

---

Thank You for advice!

---

Fixed.